src/storage-types/src/collections.proto

```diff
@@ -19,6 +19,7 @@ message GlobalId {
     uint64 user = 2;
     uint64 transient = 3;
     google.protobuf.Empty explain = 4;
+    uint64 introspection_source_index = 5;
   }
 }
```

Comment:
We actually have 3, but no I don't.

Reply:
We have one in the `catalog` protobuf objects so types we write for the catalog are isolated from the rest of the system, then another in `repr` for the rest of the system. No idea why we have this one in `storage-types` though, it can probably be deleted?